_posts/2021-12-23-Fortran-lang-2021-in-review.md

```diff
@@ -0,0 +1,355 @@
+
+You can read in more detail about their projects
+[here](https://summerofcode.withgoogle.com/archive/2021/organizations/6542461173760000).
+
+A big thank you to all students, mentors, and administrators, for their great
+work, and of course, to the Google Summer of Code program for making
+possible for students to participate.
+
+We plan to apply for Google Summer of Code in 2022 as well.
+The program is no longer limited to students and anybody 18 or older can apply
+to work on an open source project and get paid for it.
+If you're interested participating in the program in 2022 with Fortran-lang,
+don't hesitate to contact us and we'll guide you toward applying.
+
+## Conferences and papers
+
+### FortranCon 2021
+
+Like in 2020, the Fortran event of this year was
+[FortranCon 2021](https://tcevents.chem.uzh.ch/event/14/), the international
+Fortran conference, held on September 23-24.
+The keynote speaker this year was [Damian Rouson](https://github.com/rouson),
+the head of the
+[Computer Languages and Systems Software (CLaSS)](https://crd.lbl.gov/divisions/amcr/computer-science-amcr/class/)
+group at the Lawrence Berkeley National Lab. 
+Fortran-lang had a dedicated session (a Fortran-lang minisymposium) on the
+second day of the conference, with the talks about specific Fortran-lang
+projects as well as the Google Summer of Code student presentations.
+FortranCon was hosted at the University of Zurich (UZH), and organized by
+[Tiziano Müller](https://github.com/dev-zero) from UZH and
+[Alfio Lazzaro](https://github.com/alazzaro) from Hewlett-Packard Enterprise.
+You can watch all FortranCon 2021 talks [here](https://www.youtube.com/playlist?list=PLeKbr7eYHjt5UaV9zQtY24oEbne9_uFni).
+
+### PackagingCon 2021
+
+Another big event for Fortran-lang was
+[PackagingCon 2021](https://packaging-con.org/), a conference dedicated to
+package management software and its community of developers.
+[Sebastian Ehlert](https://github.com/awvwgk) presented fpm--you can watch the
+recording below:
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YG8zEM1lAVM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</center>
+
+### Papers
+
+This year we also wrote our first paper documenting the motivation and goals of
+Fortran-lang: "Toward Modern Fortran Tooling and a Thriving Developer
+Community" by Milan Curcic, Ondřej Čertík, Brad Richardson, Sebastian Ehlert,
+Laurence Kedward, Arjen Markus, Ivan Pribec, and Jérémie Vandenplas
+([https://arxiv.org/abs/2109.07382](https://arxiv.org/abs/2109.07382)).
+
+<center>
+<a href="https://arxiv.org/abs/2109.07382">
+<img width="600" src="{{ site.baseurl }}/assets/img/Fortran-lang-2021-in-review/fortran-lang-paper.png">
+</a>
+</center>
+
+If you use any of the Fortran-lang projects for your publications,
+please cite this paper as:
```

Comment:
I searched our discourse and couldn't find a post inviting the community to participate int his paper (if it is there, please let me know!). I can't remember right now how we did it. But assuming we did not invite the wider community to participate, then I would reword this part as below.

```suggestion
We have written several papers about our efforts. The main one is still in progress, we have publicly [invited](https://fortran-lang.discourse.group/t/fortran-lang-community-paper/1232) anybody in the community to participate, and once published, it will become our main paper to cite.

In the meantime, if you want, you can cite the following paper if you use any of the Fortran-lang projects for your publications:
```

Reply:
We mention the CiSE paper below, so I'd move this there. I think it's a great idea to mention the invite for the wider community. For the first paper, we agreed to not do so for pragmatic reasons.

I don't want to refer to one paper as the main one and to the other as less main. The first paper focuses on Fortran-lang projects. The second paper describes the current state of Fortran and has a section on Fortran-lang. They both stand well on their own and have different scopes. The first paper actually goes to much greater detail on Fortran-lang projects than the second one. But the second one will be in CiSE and will have more weight.

So, rather than communicating that one supersedes the other, we should encourage citing both papers.

What do you think about this, @certik @LKedward @awvwgk @jvdp1?

I will propose some edits for this.

Reply:
What do you think about these two edits:

* https://github.com/fortran-lang/fortran-lang.org/pull/355/files#r776413171
* https://github.com/fortran-lang/fortran-lang.org/pull/355/files#r776414439

I hope it's an acceptable compromise. :)

Reply:
The main issue I have is that if we did not invite the wider community for the first paper (for pragmatic reasons to get it done quickly), we should not be "requiring" everybody to cite it, since I feel the credit is not properly distributed to the many people that are helping with fortran-lang projects and are not part (and were not given a chance to be a part) of the first paper. The second paper is fundamentally different: we have invited anybody to participate, and thus we can treat it as a paper to cite.

What I want to avoid is for the wide fortran-lang contributors to be discouraged to contribute if only a small fraction is getting the "credit" in form of citations.

For that reason, reformulating these paragraphs in a way to "offer" the paper as an option to cite, but not "require" it seems much more inclusive to me.

Reply:
Ah, I see, I didn't understand that was an issue.

I agree that we absolutely should not requiring it (and we can't). Let's reword it so that it's clear. Would it read less as required to you if it was written like this:

> Please consider citing this paper if Fortran-lang projects have been helpful in your work:

Reply:
Ondřej and I spoke on the phone about this and agreed to use more careful and neutral language around the papers and citing.